### PR TITLE
Insert symbol extension

### PR DESCRIPTION
**Pull request by mstratiev**

### CODE REVIEW

File: insert-symbol/insert-symbol.provider.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.k-icon.k-i-insertsymbol {
+    font-family: WebComponentsIcons;
+    font-size: 1.4em;
+}
+
+.k-i-insertsymbol::before {
+    font-family: WebComponentsIcons;
+    content: "Ω";
```
Comment (JenPeleva) [inline insert-symbol/insert-symbol.provider.css]: Can we use the css entity for Omega here  - \03a9 :)

Comment (mstratiev) [reply insert-symbol/insert-symbol.provider.css]: is that the WebComponentsIcons symbol pointer?

**Comment by Novkirishki** [comment]
I would suggest a small change in the structure of the extensions to be done in order to be more clear and intuitive for clients:
1. Rename the **"toolbar-extender"** folder to  **"editor-extender"**.
2. Move all editor extensions in that folder - _editor config provider, videos toolbar item provider, switch text direction provider and insert symbol provider_. Each separate extension should be in its own folder. 
3. Rename **ToolbarExtenderModule** to **EditorExtenderModule** and register all of the above extensions there

File: insert-symbol/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Insert symbol
```
Comment (jordanilchev) [inline insert-symbol/README.md]: 👍

File: insert-symbol/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+## Summary
+This is an extension to the rich text editor in **Iris** that allows the user to insert a special character picked from a predefined table. The table is replaceable and could be modified to the needs of the user.
```
Comment (vassildinev) [inline insert-symbol/README.md]: Admin app instead of Iris?

Comment (mstratiev) [reply insert-symbol/README.md]: readme will be rewritten completely

File: insert-symbol/symbol-list/insert-symbol-generator.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+export const DATA_ATTRIBUTE_NAME = "data-value";
+
+/**
+ * Represents a helper class for importing a symbol list JSON and  generating html to populate the tool's popup.
+ */
+export class InsertSymbolGenerator {
+    private symbolConfig;
+
+    constructor () {
+        this.symbolConfig = require("./symbol-list.json");
```
[the remark this answers is not in the document]
Comment (mstratiev) [reply insert-symbol/symbol-list/insert-symbol-generator.ts]: Done and done

File: insert-symbol/insert-symbol.provider.ts [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+        const insertSymbolTool = ToolBuilder.createPopupTool(kendo, TOOLBAR_BUTTON_DATA.DEFAULT.name, config, null, popupTemplateGenerator);
+
+        ToolBuilder.registerTool(kendo, insertSymbolTool, TOOLBAR_BUTTON_DATA.DEFAULT.name, buttonTemplate, true);
+    }
+}
+
+export const INSERT_SYMBOL_PROVIDER: ClassProvider = {
+    multi: true,
+    provide: TOOLBARITEMS_TOKEN,
```
[the remark this answers is not in the document]
Comment (mstratiev) [reply insert-symbol/insert-symbol.provider.ts]: I have made the renaming and reordering of all the editor extensions. Please review the commit thoroughly to make sure that everything is in order now.

**Comment by mstratiev** [comment]
Please review repo restructuring to doublecheck

**Comment by Novkirishki** [comment]
Nice!

**Comment by mstratiev** [comment]
when testing is ready, it is ready for merge

File: editor-extender/insert-symbol/symbol-list/symbol-list.json [hunk 6]
```diff
@@ -0,0 +1 @@
..."name":"⊆","value":"⊆","escapedValue":"&amp;sube; &amp;SubsetEqual; &amp;subseteq;","tooltip":"SUBSET OF OR EQUAL TO"},{"name":"⊇","value":"⊇","escapedValue":"&amp;supe; &amp;supseteq; &amp;SupersetEqual;","tooltip":"SUPERSET OF OR EQUAL TO"},{"name":"⊕","value":"⊕","escapedValue":"&amp;oplus; &amp;CirclePlus;","tooltip":"CIRCLED PLUS"},{"name":"⊗","value":"⊗","escapedValue":"&amp;otimes; &amp;CircleTimes;","tooltip":"CIRCLED TIMES"},{"name":"⊥","value":"⊥","escapedValue":"&amp;bottom; &amp;bot; &amp;perp; &amp;UpTee;","tooltip":"UP TACK"},{"name":"⌈","value":"⌈","escapedValue":"&amp;lceil; &amp;LeftCeiling;","tooltip":"LEFT CEILING"},{"name":"⌉","value":"⌉","escapedValue":"&amp;rceil; &amp;RightCeiling;","tooltip":"RIGHT CEILING"},{"name":"⌊","value":"⌊","escapedValue":"&amp;lfloor; &amp;LeftFloor;","tooltip":"LEFT FLOOR"},{"name":"⌋","value":"⌋","escapedValue":"&amp;rfloor; &amp;RightFloor;","tooltip":"RIGHT FLOOR"},{"name":"⟨","value":"⟨","escapedValue":"&amp;lang; &amp;LeftAngleBracket; &amp;langle;","tooltip":"MATHEMATICAL LEFT ANGLE BRACKET"},{"name":"⟩","value":"⟩","escapedValue":"&amp;rang; &amp;RightAngleBracket; &amp;rangle;","tooltip":"MATHEMATICAL RIGHT ANGLE BRACKET"}]
```
Comment (dechoD) [inline editor-extender/insert-symbol/symbol-list/symbol-list.json]: Can we beautify this json?

File: editor-extender/insert-symbol/symbol-list/symbol-list.full.json [hunk 7]
```diff
@@ -0,0 +1 @@
...","tooltip":"SINE WAVE"},{"name":"≀","value":"≀","escapedValue":"&amp;wreath; &amp;VerticalTilde; &amp;wr;","tooltip":"WREATH PRODUCT"},{"name":"≁","value":"≁","escapedValue":"&amp;nsim; &amp;NotTilde;","tooltip":"NOT TILDE"},{"name":"≂","value":"≂","escapedValue":"&amp;esim; &amp;EqualTilde; &amp;eqsim;","tooltip":"MINUS TILDE"},{"name":"≃","value":"≃","escapedValue":"&amp;sime; &amp;TildeEqual; &amp;simeq;","tooltip":"ASYMPTOTICALLY EQUAL TO"},{"name":"≄","value":"≄","escapedValue":"&amp;nsime; &amp;nsimeq; &amp;NotTildeEqual;","tooltip":"NOT ASYMPTOTICALLY EQUAL TO"},{"name":"≅","value":"≅","escapedValue":"&amp;cong; &amp;TildeFullEqual;","tooltip":"APPROXIMATELY EQUAL TO"},{"name":"≆","value":"≆","escapedValue":"&amp;simne;","tooltip":"APPROXIMATELY BUT NOT ACTUALLY EQUAL TO"},{"name":"≇","value":"≇","escapedValue":"&amp;ncong; &amp;NotTildeFullEqual;","tooltip":"NEITHER APPROXIMATELY NOR ACTUALLY EQUAL TO"},{"name":"≈","value":"≈","escapedValue":"&amp;asymp; &amp;ap; &amp;TildeTilde; &amp;approx; &amp;thkap; &amp;thickapprox;","tooltip":"ALMOST EQUAL TO"},{"name":"≉","value":"≉","escapedValue":"&amp;nap; &amp;NotTildeTilde; &amp;napprox;","tooltip":"NOT ALMOST EQUAL TO"},{"nam
```
Comment (dechoD) [inline editor-extender/insert-symbol/symbol-list/symbol-list.full.json]: Can we beautify this file?

**Pull request merged**